cores/xmc_lib/XMCLib/inc/xmc1_scu.h

```diff
@@ -137,7 +137,7 @@
 #define XMC_SCU_INTERRUPT_EVENT_TSE_DONE         SCU_INTERRUPT_SRMSK_TSE_DONE_Msk  /**< Temperature measurement Completion event. */
 #define XMC_SCU_INTERRUPT_EVENT_TSE_HIGH         SCU_INTERRUPT_SRMSK_TSE_HIGH_Msk  /**< Temperature too high event. */
 #define XMC_SCU_INTERRUPT_EVENT_TSE_LOW          SCU_INTERRUPT_SRMSK_TSE_LOW_Msk /**< Temperature too low event. */
-#if defined(CAN)
+#if defined(CAN_xmc)
```

Comment:
This should be a patched in the future from a xmclib package.

Reply:
I will add new ticket